src/ontology/mondo-ingest.Makefile

```diff
@@ -343,6 +343,17 @@ lexmatch/README.md: $(SCRIPTSDIR)/lexmatch-sssom-compare.py ../mappings/mondo-so
 
 extract-unmapped-matches: lexmatch/README.md
 
+#############################
+######### Analysis ##########
```

Comment:
@matentzn I made a section in the `makefile` called "Analysis" and added the goal there. I also put the Python scripting in `src/analysis/`.

There is one other script in `src/analysis`, but it's specific to exclusions, so I left that one under the "exclusions" section instead of here.